test/runScript-mock-findBin.spec.js

```diff
@@ -4,22 +4,20 @@
 
 import mockFn from 'execa'
 import runScript from '../src/runScript'
+import findBinMock from '../src/findBin'
 
 jest.mock('execa')
+jest.mock('../src/findBin')
 
 // Mock findBin to return an absolute path
-jest.mock(
-  '../src/findBin',
-  () => commands => {
-    const [bin, ...otherArgs] = commands.split(' ')
+findBinMock.mockImplementation(commands => {
```

Comment:
Since the mock implementation never changes, should this go to `__mocks__` similar to execa and npm-which?

Reply:
If we wanted to do that for a source file, the mock would need to be placed in a directory `__mocks__` inside `src`. Since it is used in only 1 file, let's keep it in the test file itself.